mercury.spec

```
@@ -91,13 +81,18 @@ Mercury plugin to support the UCX transport.
 %autosetup -p1 -n mercury-%dl_version
 
 %build
+# Pass -pie linker flags manually on CentOS7 due to CMake PIE issues
+# see: https://cmake.org/cmake/help/latest/policy/CMP0083.html#policy:CMP0083
 %cmake  -DCMAKE_IN_SOURCE_BUILD:BOOL=ON                   \
         -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo          \
+%if 0%{?rhel} < 8
```

This will match non EL platforms also, such as SUSE, etc.

```suggestion
%if 0%{?rhel} && 0%{?rhel} < 8
```

---

ah good catch, thanks!